### DISCUSSION TITLE
Issue with Github pages deployment

### DISCUSSION THREAD

**Discussion by shwetankverma** [community/community#59938] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Is there any issue with the github pages deployment. My workflows are getting queued or running forever

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @shwetankverma ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by ilanvaks** [comment]
Did u deploy correctly? What steps have you taken so far? Seems to be working fine for me at the moment.
